src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -1842,6 +1842,17 @@ public static class ProfileOptimization
         public static void StartProfile(string profile) { throw null; }
     }
 }
+namespace System.Runtime.CompilerServices
+{
+    [Serializable]
+    public sealed class SwitchExpressionException : System.InvalidOperationException
+    {
+        public SwitchExpressionException() { }
+        public SwitchExpressionException(object unmatchedValue) { }
+        public object UnmatchedValue { get { throw null; } }
+        public override void GetObjectData(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
```

Comment:
I believe, despite what it says in the issue, this can be protected instead of public override. (Compare other exceptions in the refs. I see 6 that are public, all in directory services.)

Reply:
> this can be protected instead of public override

No, it needs to be public.  You can't narrow visibility when overriding, and the base virtual is public.

Reply:
You're right. I must have been somehow looking at the serialization constructors.